opshin/type_inference.py

```diff
@@ -838,6 +838,24 @@ def visit_Attribute(self, node: Attribute) -> TypedAttribute:
     def visit_Assert(self, node: Assert) -> TypedAssert:
         ta = copy(node)
         ta.test = self.visit(node.test)
+        try:
+            warn_assert_msg = f" (see assert with message '{node.msg.value}')"
+        except Exception:
+            warn_assert_msg = ""
```

Comment:
I agree, this should likely be a TypeError or AttributeError (you should be able to print the error at runtime since it would otherwise be masked as syntaxerror)